.github/workflows/release.yaml

```diff
@@ -64,6 +64,22 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.LIT_ROBOT_AUTOMATION_PAT }}
           NPM_TOKEN: ${{ secrets.LIT_ROBOT_NPM_PUBLISH_ACCESS_TOKEN }}
 
+      - name: Build release-image script
+        if: steps.cs.outputs.published == 'false'
+        working-directory: lit/packages/internal-scripts
+        run: npm run build
+
+      - name: Generate release image artifact
+        if: steps.cs.outputs.published == 'false'
+        with:
+          cwd: lit
+        run: |
+          $(node scripts/fetch-release-changelog.js <<< ${{ steps.cs.outputs.pullRequestNumber }})
```

Comment:
It's unfortunate that we have to fetch this separately when the versions action above already has it. This does work but I wonder if we might either make it a feature request to output the pr body or make a fork? https://github.com/changesets/action/blob/6a46d2c28377497e64763ba5f1faeabbd3d88c33/src/run.ts#L367-L369

This is probably more of a niche use case though.. it may be more effort than we want.

Reply:
Agree. Once this approach is tested and I have more confidence in it I can file an issue with the feature request.